packages/ses/scripts/bundle.js

```diff
@@ -81,6 +101,10 @@ const main = async () => {
   console.log(`Copied ${sourceDTS} to ${destDTS}`);
 };
 
+const main = async () => {
+  await writeBundle({ buildType: process.env.SES_BUILD_TYPE || 'vanilla' });
+};
```

Comment:
the env var could refactored to an arg instead if preferred, i.e.
before: `SES_BUILD_TYPE=hermes node scripts/bundle.js`
after: `node scripts/bundle.js hermes`

Reply:
Please use the env-options package rather than accessing process.env manually

Reply:
updated ^ https://github.com/endojs/endo/pull/2334/commits/5dc71f913dc4a753334451d59bbb8591f49da769

https://github.com/endojs/endo/blob/5dc71f913dc4a753334451d59bbb8591f49da769/packages/ses/scripts/bundle.js#L105-L108